colcon_bundle/verb/_archive_generators.py

```diff
@@ -84,10 +83,7 @@ def generate_archive_v2(install_base,
     |- dependencies.tar.gz
     |- workspace.tar.gz
 
-    :param install_base: Directory with built artifacts from the workspace
-    :param dependencies_staging_path: Directory where all dependencies
-    have been installed
-    :param bundle_base: Directory to place the output of this function
+    :param bundle_path: BundlePath object including all path configurations
```

Comment:
Calling the object 'BundlePath' is a bit confusing since the object isn't actually a path to a bundle. Can we call it 'PathContext'?